pkg/cmd/datamgr/cli/server/server.go

```diff
@@ -156,6 +155,7 @@ type server struct {
 	dataMover             *dataMover.DataMover
 	snapManager           *snapshotmgr.SnapshotManager
 	externalDataMgr       bool
+	vcConfigSecret		  bool
```

Comment:
gofmt?

Can you run gofmt in all the directories that you have made changes?

Reply:
done